modules/nf-core/sentieon/dnascope/main.nf

```diff
@@ -0,0 +1,88 @@
+process SENTIEON_DNASCOPE {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    container 'nf-core/sentieon:202112.06'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+    tuple val(meta4), path(dbsnp)
+    tuple val(meta5), path(dbsnp_tbi)
+    tuple val(meta6), path(call_interval)
+    path ml_model
+
+    output:
+    tuple val(meta), path("*.vcf.gz")                      , emit: vcf
+    tuple val(meta), path("*.vcf.gz.tbi")                  , emit: index
+    tuple val(meta), path("*.vcf.gz"), path("*.vcf.gz.tbi"), emit: vcf_index
```

Comment:
I think it is a bit unconventional to have the vcf and the index-files combined in one output-channel. Perhaps we should ask on Slack what is considered best-practice?

Reply:
I don't have any strong preferences tbh. We can go with your suggestion for now, as that seems to be the standard. We can bring this up in the next maintainers meeting, and see what others think 😄